### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by jedmao**
Playing w/ Travis a bit here too.

### CODE REVIEW

**Comment by jedmao** [comment]
@SamVerschueren, published as v0.3.4.

File: src/DocumentWatcher.ts [hunk 1]
```diff
@@ -177,7 +177,7 @@ function applyEditorConfigToTextEditor(
 	);
 
 	/* tslint:disable:no-any */
-	textEditor.options = <any> newOptions;
+	textEditor.options = newOptions as any;
```
Comment (SamVerschueren) [inline src/DocumentWatcher.ts]: Cool, is this new in TS@2.x?

**Comment by SamVerschueren** [comment]
🎉  Great work @jedmao !

File: src/DocumentWatcher.ts (hunk 1 above)
Comment (jedmao) [inline src/DocumentWatcher.ts]: Yeah. I read about it recently, because it's the only syntax that works in a TSX file for React in some scenarios.

**Comment by jedmao** [comment]
Good to know. I'll put that on the backburner.

**Pull request merged**